code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1524,6 +1524,22 @@
 	REMOVE_TRAIT(L, TRAIT_RESISTHEAT, type)
 	return ..()
 
+/datum/reagent/zauker
+	name = "Zauker"
+	description = "An unstable gas that is toxic to all living beings"
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "90560B"
+	taste_description = "bitter"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED|REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/zauker/on_mob_life(mob/living/L, delta_time, times_fired)
+	. = ..()
+	L.adjustBruteLoss(25 * REM * delta_time, FALSE)
+	L.adjustOxyLoss(5 * REM * delta_time, FALSE)
+	L.adjustFireLoss(8 * REM * delta_time, FALSE)
+	L.adjustToxLoss(8 * REM * delta_time, FALSE)
```

Comment:
Don't let this be synthesizable or else botany will have a field day

Reply:
I think if botany manages to roll this, they deserve their lottery

Reply:
Would this mean people could remove the reagent from someone's bloodstream and use it as poison?